diff --git a/node_modules/react-native-calendar-strip/src/CalendarDay.js b/node_modules/react-native-calendar-strip/src/CalendarDay.js
index 930b0fd..68c6f1b 100644
--- a/node_modules/react-native-calendar-strip/src/CalendarDay.js
+++ b/node_modules/react-native-calendar-strip/src/CalendarDay.js
@@ -396,6 +396,8 @@ class CalendarDay extends Component {
       dateNumberFontSize,
     } = this.state;
 
+ 
+
     let _dateNameStyle = [styles.dateName, enabled ? dateNameStyle : disabledDateNameStyle];
     let _dateNumberStyle = [styles.dateNumber, enabled ? dateNumberStyle : disabledDateNumberStyle];
     let _dateViewStyle = enabled
@@ -486,7 +488,8 @@ class CalendarDay extends Component {
               styles.dateContainer,
               responsiveDateContainerStyle,
               _dateViewStyle,
-              containerStyle
+              containerStyle,
+              {marginTop:selected ?25:15}
             ]}
           >
             {showDayName && (
diff --git a/node_modules/react-native-calendar-strip/src/Scroller.js b/node_modules/react-native-calendar-strip/src/Scroller.js
index a612538..6dcc6fe 100644
--- a/node_modules/react-native-calendar-strip/src/Scroller.js
+++ b/node_modules/react-native-calendar-strip/src/Scroller.js
@@ -42,6 +42,7 @@ export default class CalendarScroller extends Component {
       const itemHeight = renderDayParams.height;
       const itemWidth = renderDayParams.width + renderDayParams.marginHorizontal * 2;
 
+
       const layoutProvider = new LayoutProvider(
         index => 0, // only 1 view type
         (type, dim) => {
@@ -334,7 +335,7 @@ export default class CalendarScroller extends Component {
 
     return (
       <View
-        style={{ height: this.state.itemHeight, flex: 1 }}
+        style={{ height: this.state.itemHeight+30, flex: 1 }}
         onLayout={this.onLayout}
       >
         <RecyclerListView
